backend/tracim_backend/command/__init__.py

```diff
@@ -97,3 +99,14 @@ def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
             default=False,
         )
         return parser
+
+    def changeDirectory(self) -> None:
+        directoryPath = os.path.dirname(__file__)
+        if len(directoryPath) > 30 and directoryPath[-30:] == "backend/tracim_backend/command":
+            os.chdir(os.path.dirname(__file__))
+            os.chdir("../..")
+        else:
+            try:
+                os.chdir("/etc/tracim")
```

Comment:
this is the folder where find config file, but for example, git is not present in this folder (git used when execute tracimcli command to show actual version of tracim).

Reply:
ok, so in with folder should we be to execute a tracimcli command on docker ?

Reply:
I think it is in `/tracim` or `/tracim/backend`